readme.md

```diff
@@ -165,21 +165,84 @@ swa start http://localhost:3000 --swa-config-location ./my-app-source
+| `--ssl`                          | serving the app and API over HTTPS (default: false)     | `false`                 | `--ssl` or `--ssl=true`                              |
+| `--ssl-cert`                     | SSL certificate to use for serving HTTPS                |                         | `--ssl-cert="/home/user/ssl/example.crt"`            |
+| `--ssl-key`                      | SSL key to use for serving HTTPS                        |                         | `--ssl-key="/home/user/ssl/example.key"`             |
+| `--run`                          | Run a command at startup                                |                         | `--run="cd app & npm start"`                         |
+| `--devserver-timeout`            | The time to wait(in ms) for the dev server to start     | 30000                   | `--devserver-timeout=60000`                          |
+| `--func-args`                    | Additional arguments to pass to `func start`            |                         | `--func-args="--javascript"`                         |
+| `--config`                       | Path to swa-cli.config.json file to use.                | `./swa-cli.config.json` | `--config ./config/swa-cli.config.json`              |
+| `--print-config`                 | Print all resolved options. Useful for debugging.       |                         | `--print-config` or `--print-config=true`            |
+
+## swa-cli.config.json file
+
+The CLI can also load options from a `swa-cli.config.json` file.
+
+```json
+{
+  "configurations": {
+    "app": {
+      "context": "http://localhost:3000",
+      "apiLocation": "api",
+      "run": "npm run start",
+      "swaConfigLocation": "./my-app-source"
+    }
+  }
+}
+```
+
+If only a single configuration is present in the `swa-cli.config.json` file, running `swa start` will use it by default. If options are loaded from a config file, no options passed in via command line will be respected. For example `swa start app --ssl=true`. The `--ssl=true` option will not be picked up by the CLI.
+
+### Example
+
+We can simplify these commands by putting the options into a config file.
+
+```bash
+# static configuration
+swa start ./my-dist --swa-config-location ./my-app-source
+
+# devserver configuration
+swa start http://localhost:3000 --swa-config-location ./my-app-source
+```
+
+```json
+{
+  "configurations": {
+    "static": {
+      "context": "./my-dist",
+      "swaConfigLocation": "./my-app-source"
+    },
+    "devserver": {
+      "context": "http://localhost:3000",
+      "swaConfigLocation": "./my-app-source"
+    }
+  }
+}
+```
+
+These configurations can be run with `swa start static` and `swa start devserver`.
+
+### Validation
+
+You can validate your `swa-cli.config.json` with a JSON Schema like so:
+
+```json
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/static-web-apps-cli/main/schema/swa-cli.config.schema.json",
```

Comment:
Suggestion/idea: host this config schema file somewhere on docs/ACOM?

cc @anthonychu @simonaco